docs/Medical/Basic_Medical.md

```diff
@@ -61,15 +61,46 @@ _How to interact with ACE Medical as a player_
+### Is the patient alive
+  This can sometimes be tricky to tell, since the unconcious state can appear very similar to the dead state. Heartrate cannot be relied on to distinguish between these states since it can be non-existent in both states.
+  The easiest way to tell if a patient is dead, and therefore beyond saving, is to look at their character model. If they are ragdolled they are dead. If they are in a wounded animation they are still alive.
 ### Is the patient conscious
+  Firstly, determine if the player is concious. This is very easy, if they are moving around and talking, they are concious. If they are in any of the unconcious animations, they are not.
+  A concious players need for medical attention is less immediate.
 ### How hurt is the patient
+  When first opening the ACE Medical menu i like to get a quick overview of roughly how bad the patients wounds are by simply looking at the colors on the player chart. Some blue and light yellow parts are to be expected but often not immedieatly lifethreatening. Alow of yellow or red parts however moves the timetable up quite a bit.
 ### Does the patient have vitals
+  When we have a rough idea of the overall state of the patient, we will check their vitals. This can be done by selecting a limb that is not tourniqueted, and checking for their blood pressure or heartrate.
+  I usually only check the bloodpressure, since no bloodpressure will also mean that there is no heartrate. Any bloodpressure will tell you both that there is heart activity, and also how their bloodlevels are doing.
+
+  When a player looses their hearbeat, ACE will start a timer, when that timer runs out the player dies. This is why its important to check the vitals early in the evaluation stage so that the timer does not run out while we are fixing other stuff.
+  The length of this timer varies based on ACE settings, but is by default set to 5 minutes. 
+
 ### How much is the patient bleeding
+  The amount of bleeding can be read out from the right panel on the ACE Medical menu. It's important to stop large bleeds quickly, since loosing too much blood will kill the player long before the heartbeat timer runs out.
+  The colors of bodyparts in the patients overview chart will tell you what is bleeding, and how badly.
+
 ### How long ago did the patient receive damage
+  This wont always be know accurately, but a rough estimate is often good enough. Knowing this will help you determine the best course of action to stop the patient from dying.
+  If the player has been down for a long time, the first thing that you should concider is the heartbeat timer. Check the vitals quickly, and if they are not there fix them before patching up bleeding.
+  If the patient has been down for either a short time, there is sufficient time to first stop bleeding and then worry about the vitals.
```

Comment:
My thinking is that anyone in the squad should be able to stablize a downed player while waiting for a medic even if they cant get them back up by themselves. So a quick overview of vitals is needed but i agree it can be pulled back to the very basics.